### PR TITLE
added save functionality

### PR DESCRIPTION
**Pull request by Suhas0723**
Me and Suchit did the save functionality to save the locations for registered users

### CODE REVIEW

**Review by NewTigerShark10 — APPROVED** [review]

File: src/Database/Location.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+namespace App\Database;
+
+use PDO;
+class Location extends Database {
+
+public function add_location($woeid, $locations, $email){
+    $query = $this->db->prepare('SELECT * from users WHERE email_address = :email_address');
+    $query -> execute([
+        ':email_address' => $email]);
+    $location = $query->fetch(PDO::FETCH_ASSOC);
```
Comment (jamiekohns) [inline src/Database/Location.php:11]: nit: `$location` should probably be `$user`

File: src/Database/Location.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    $location_query = $this->db->prepare('INSERT into location (woeid, location, user_id) VALUES (:woeid, :location, :user_id)');
+    $location_query->execute([
+      ':woeid' => $woeid,
+      ':location' => $locations,
+      ':user_id' => $location['id'],]);
```
Comment (jamiekohns) [inline src/Database/Location.php:16]:
```
[
      ':woeid' => $woeid,
      ':location' => $locations,
      ':user_id' => $location['id'],
]
```

File: test.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?php
```
Comment (jamiekohns) [inline test.php:1]: remove `test.php`

**Review by jamiekohns — APPROVED** [review]
Cleanup nits and remove test.php, then it's gtg

**Pull request merged**